### ISSUE TITLE
import 'foo'

### ISSUE THREAD

**Issue by rubas** [jcschmidig/snowpack-resolve-alias#1]
Thanks for the plugin. Works well except it doesn't seem to catch imports like following. 

```
import 'foo'
```

I have to add `foo` manually to the webpack config.

```
config.resolve = {
alias: {
  foo: path.resolve(out, 'foo')
 }
}
```

**Comment by jcschmidig** [comment]
How does the section
alias: { ... } look like in snowpack.config.json?

**Comment by rubas** [comment]
All other defined aliases work fine. But it is also the only one that is is imported with a simple `import 'foo'` .

If I add it manually to the webpack config it works too. 

```
const root = 'baar/baz'

/** @type {import("snowpack").SnowpackUserConfig } */
module.exports = {
  alias: {
    foo: path.resolve(root, 'foo'),
  },
```

**Comment by jcschmidig** [comment]
Sorry I don't get the problem. Could you give me
- the _complete_ section **alias** of the snowpack.config
- the error you get during the build

Thanks!

**Comment by rubas** [comment]
Thanks for the feedback. Let me be more specific.

This code works fine.
`import { userIsLoggedIn } from '@user'`

This doesn't work for (webpack) build. It will not be replaced with the alias.
`import  '@user'`

## Workaround
I have to add the **same** alias for `@user` "manually" to webpack config via the extendedConfig parameter of `@snowpack/plugin-webpack`.

**Comment by jcschmidig** [comment]
Thanks, now I got it. Let me look into this!

**Comment by jcschmidig** [comment]
I wasn't aware of this use of import...
Fixed it in version 1.2.8
